docs/content/querying/segmentmetadataquery.md

```diff
@@ -49,6 +49,9 @@ The format of the result is:
   "aggregators" : {
     "metric1" : { "type" : "longSum", "name" : "metric1", "fieldName" : "metric1" }
   },
+  "queryGranularity" : {
+    "type": "all"
```

Comment:
you could do a batch ingestion and have segment interval be min instant to max instant.

but, changed doc to "none" as that is more obvious.